cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -121,10 +119,6 @@ public BufferingTable(String name, ConflictDetection level, boolean enableReadle
     this.name = name;
     this.conflictLevel = level;
     this.enableReadlessIncrements = enableReadlessIncrements;
-    // TODO: having central dataset management service will allow us to use table ids instead of names, which will
```

Comment:
dont really understand what this means, but do we want to preserve this comment?